account_credit_control/__manifest__.py

```diff
@@ -5,7 +5,7 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 {
     "name": "Account Credit Control",
-    "version": "16.0.1.0.2",
+    "version": "16.0.1.0.3",
```

Comment:
Don't change manually the manifest version. It will be done on merge.

Reply:
Done.

Reply:
@pedrobaeza I squashed the commits and action/checks are successful.